cluster/gce/upgrade.sh

```diff
@@ -439,6 +439,75 @@ function do-node-upgrade() {
   echo "== Finished upgrading nodes to ${KUBE_VERSION}. ==" >&2
 }
 
+CURRENT_COREDNS_VERSION=$(${KUBE_ROOT}/cluster/kubectl.sh -n kube-system get deployment coredns -o=jsonpath='{$.spec.template.spec.containers[:1].image}' | cut -d ":" -f 2)
```

Comment:
This line is hard to notice --- can we move this down to after all the funcs definition? And logging around this might be helpful when debugging.

Reply:
done